### PR TITLE
fix(modify:fluid-imports): Use narrower checks for whether an import is a Fluid import

### PR DESCRIPTION
**Pull request by tylerbutler**
This change will ensure that packages from scopes like `@fluidx` will not be treated as framework packages.

### CODE REVIEW

File: build-tools/packages/build-cli/src/commands/modify/fluid-imports.ts [hunk 1]
```diff
@@ -517,7 +517,11 @@ function parseFluidImports(
 }
 
 function isFluidImport(packageName: PackageName): boolean {
-	return packageName.startsWith("@fluid") || unscopedFFPackages.has(packageName);
+	return (
```
Comment (Josmithr) [inline build-tools/packages/build-cli/src/commands/modify/fluid-imports.ts:520]: Nit: probably worth adding some notes here about why the narrow checks are needed.

**Review by Josmithr — APPROVED** [review]

**Review by sonalideshpandemsft — APPROVED** [review]

File: build-tools/packages/build-cli/src/commands/modify/fluid-imports.ts (hunk 1 above)
Comment (jason-ha) [reply build-tools/packages/build-cli/src/commands/modify/fluid-imports.ts:520]: And we'll want to remember if we every deviate from any of those.

Comment (tylerbutler) [reply build-tools/packages/build-cli/src/commands/modify/fluid-imports.ts:543]: Added some docs, but I also switched to using a fixed list of known scopes.

File: build-tools/packages/build-cli/src/commands/modify/fluid-imports.ts [hunk 2]
```diff
@@ -15,10 +15,27 @@ import type { CommandLogger } from "../../logging";
...
+	"@fluidframework",
+	"@fluid-example",
+	"@fluid-experimental",
+	"@fluid-internal",
+	"@fluid-private",
+	"@fluid-tools",
+] as const;
+
 /**
  * FF packages that exist outside of a scope that starts with `@fluid`.
```
Comment (jason-ha) [inline build-tools/packages/build-cli/src/commands/modify/fluid-imports.ts:36]: Comment slightly off. ;)

**Pull request merged**